core/blockchain.go

```diff
@@ -1053,6 +1055,21 @@ func (bc *BlockChain) Accept(block *types.Block) error {
 	bc.addAcceptorQueue(block)
 	acceptedBlockGasUsedCounter.Inc(int64(block.GasUsed()))
 	acceptedTxsCounter.Inc(int64(len(block.Transactions())))
+	if baseFee := block.BaseFee(); baseFee != nil {
+		blockGasPriceGauge.Update(baseFee.Int64())
+	}
+
+	totalFees := big.NewInt(0)
+	for _, tx := range block.Transactions() {
+		gasFeeCap := tx.GasFeeCap()
+		gasTipCap := tx.GasTipCap()
+		gasUsed := new(big.Int).SetUint64(tx.Gas())
```

Comment:
I could be wrong.. but I believe that `tx.Gas()` returns the gas supplied and the gas used is available in receipts. eg:
https://github.com/ava-labs/coreth/blob/master/miner/worker.go#L370
I think we should follow the same computation with `tx.EffectiveGasTipValue(baseFee)` as well.

Reply:
That's correct